rtl/eclrtl/rtldynfield.cpp

```diff
@@ -38,9 +38,9 @@ extern ECLRTL_API RecordTranslationMode getTranslationMode(const char *val)
 {
     if (!val || strToBool(val) || strieq(val, "payload"))
         return RecordTranslationMode::Payload;
-    else if (strieq(val, "alwaysDisk"))
+    else if (strieq(val, "alwaysDisk") || strieq(val, "disk"))
```

Comment:
Do we really need both?

Reply:
no, but first is for backward compatibility, and second seems more logical to me.

Reply:
If 7.0 only, then I think we should settle on just "disk" and "ecl"